### PR TITLE
Make Universal

### LINKED ISSUE #459: Migrate to universal runtime

### PR DESCRIPTION
**Pull request by trieloff**
fixes #459

### CODE REVIEW

File: src/sendquery.js [hunk 1]
```diff
@@ -140,20 +141,16 @@ async function execute(email, key, project, query, service, params = {}) {
  * get query metadata
  * @param {object} params parameters for substitution into query
  */
-async function queryInfo(params) {
-  const path = params.__ow_path.split('.')[0];
+async function queryInfo(pathname, params) {
+  const [path] = pathname.split('.');
```
Comment (dominique-pfister) [inline src/sendquery.js:145]: so a `pathname` will never contain a `.` in a previous segment, or are you only interested in the path *upto* that `.` anyway?

**Review by dominique-pfister — APPROVED** [review]

File: src/sendquery.js (hunk 1 above)
Comment (trieloff) [reply src/sendquery.js:145]: And clearer, too. The various substring methods and their respective indices are just too hard to remember.

Comment (dominique-pfister) [reply src/sendquery.js:145]: True 😄

**Review by tripodsan — APPROVED** [review]

**Pull request merged**